system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -12,13 +12,34 @@
    [cmr.system-int-test.data2.core :as d]
    [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
    [cmr.system-int-test.data2.umm-spec-common :as data-umm-cmn]
+   [cmr.system-int-test.utils.humanizer-util :as hu]
```

Comment:
Nitpick: we're getting away from abbreviating requires like this - something like `humanizer-util` would be preferred